src/main/java/picard/arrays/illumina/IlluminaBPMLocusEntry.java

```diff
@@ -0,0 +1,79 @@
+package picard.arrays.illumina;
+
+/**
+ * A simple class to represent a locus entry in an Illumina Bead Pool Manifest (BPM) file
+ */
+public class IlluminaBPMLocusEntry {
```

Comment:
Consider passing the InfiniumDataFile into the constructor and parsing there

Reply:
I'm going to hold off on that for now.  Maybe in another PR